src/mixins/legal-api-mixin.ts

```diff
@@ -75,6 +75,29 @@ export default class LegalApiMixin extends Vue {
       })
   }
 
+  /**
+   * Fetches a filing.
+   * @returns a promise to return the filing of the specified type, or null if not found
+   */
+  async fetchFiling (businessId: string, filingType: string): Promise<IncorporationFilingIF> {
+    const url = `businesses/${businessId}/filings`
+    return axios.get(url)
+      .then(response => {
+        const filings = response?.data?.filings
+        const returnfiling = filings.find(filings => filings.filing.header.name === filingType)
```

Comment:
`filings` could be null/undefined -- use ?

Reply:
ahh, yes will update this!